basilisk/src/ast/interpreter/dimension-tests/test1.c

```
@@ -0,0 +1,31 @@
+/**
+# "Undefined branched expressions" have the same dimensions */
+
+int main()
+{
+  init_grid (1);
+  {
+    double a, b;
+
+    if (a)
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)